runtime/android/core/src/org/xwalk/core/XWalkCoreWrapper.java

```diff
@@ -222,6 +234,25 @@ private boolean findEmbeddedCore() {
         return true;
     }
 
+    private boolean findExpansionCore() {
+        mBridgeContext = new XWalkExpansionContext(mWrapperContext);
```

Comment:
Can you please double check if it works in Android older than JELLY_BEAN_MR1? Since you assign a non-null mBridgeContext, checkCoreArchitecture() -> XWalkViewDelegate.loadXWalkLibrary() will try to load a libxwalkcore.so from /data/data/" + mBridgeContext.getPackageName() + "/lib"  if Android version < JELLY_BEAN_MR1